R/snapshot-file.R

```diff
@@ -21,6 +21,17 @@
 #' @param binary If `FALSE`, files are compared line-by-line, ignoring the
 #'   difference between Windows and Mac/Linux line endings.
 #' @inheritParams expect_snapshot
+#'
+#' @section Announcing snapshots:
+#' testthat automatically detects dangling snapshots that have been
+#' written to the `_snaps` directory but which no longer have
+#' corresponding R code to generate them. These dangling files are
+#' automatically deleted so they don't clutter the snapshot
+#' directory. However we want to preserve snapshot files when the R
+#' code wasn't executed because of an unexpected error or because of a
+#' [skip()]. Let testthat know about these files by calling
+#' `announce_snapshot_file()` before `expect_snapshot_file()`.
```

Comment:
I think you should also add `expect_snapshot_png()` in the example below to show what a full wrapper will look like. Otherwise LGTM.